apps/src/music/player/MusicLibrary.ts

```diff
@@ -413,6 +415,10 @@ export interface SoundData {
   skipLocalization?: boolean;
 }
 
+interface SoundDataInstrument extends Omit<SoundData, 'length' | 'type'> {}
```

Comment:
Thanks for adding this! Can we also add an required `note` prop here so it overrides the optional `note` prop in `SoundData`?

Edit: we could also go farther and trim all these interfaces down to only the keys that the actually use? for example, removing `note` from `SoundData` and removing `restricted`, `bpm`, `key`, etc from `SoundDataInstrument/Kit`